lib/network_config.ml

```diff
@@ -90,8 +90,12 @@ let read_config () =
 	try
 		let config_json = Unixext.string_of_file config_file_path in
 		config_json |> Jsonrpc.of_string |> config_t_of_rpc
-	with e ->
-		error "Error while trying to read networkd configuration: %s\n%s"
-			(Printexc.to_string e) (Printexc.get_backtrace ());
-		raise Read_error
+	with
+		| Unix.Unix_error (Unix.ENOENT, _, file) ->
+			error "Cannot read networkd configuration file %s because it does not exist." file;
+			raise Read_error
+		| _ as e ->
```

Comment:
Yes, it's the same when I tested. But I was wondering what would happen if some variable or type in the future were created named `e` and then it matched here accidentally. I am being overly-explicit out of caution, or is there no need as the case I've described can't happen?